ios/PolyPodApp/PolyPod/RuntimeInfo.swift

```diff
@@ -11,4 +11,45 @@ struct RuntimeInfo {
         let buildNumber = info["CFBundleVersion"] ?? "Unknown"
         return "\(marketingVersion) (\(buildNumber))"
     }
+    
+    static var isProduction: Bool {
+        if isDebug || isRunningInTestFlightEnvironment {
+            return false
+        }
+        
+        return true
+    }
+    
+    static var isDebug: Bool {
```

Comment:
Can be private, hm?

Reply:
Could be, left it as internal so it can be used for other needs.